### PR TITLE
Disable USB wakeup on the ASUS ITE 8295 Device on Asus Zephyrus GA402X laptop

### PR DESCRIPTION
**Pull request by mexisme**
###### Description of changes

This allows a user to disable the `power/wakeup` on the ASUS ITE 8245 Device Vendor `0b05`, Model `193b`.
This device tends to send a power "wakeup" immediately after a Suspend, meaning the Suspend  is resumed before it can actually suspend.

This is set to disable the wakeup, by default.

###### Things done

- [x] Tested the changes in your own NixOS Configuration
- [x] Tested the changes end-to-end by using your fork of `nixos-hardware` and
      importing it via `<nixos-hardware>` or Flake input

### CODE REVIEW

**Comment by Mic92** [comment]
@mergify queue

File: asus/zephyrus/ga402x/shared.nix [hunk 1]
```diff
@@ -32,6 +32,12 @@ in {
     keyboard.autosuspend.enable = (
       mkEnableOption "Enable auto-suspend on the internal USB keyboard (ASUS N-KEY Device) on Zephyrus GA402X"
     ) // { default = defaultAutosuspendEnable; };
+    # The ASUS 8295 ITE device will cause an immediate wake-up when trying to suspend the laptop.
+    # After the first successful hibernate, it will work as expected, however.
+    # NOTE: I'm not actually sure what this device, as neither the touchpad nor the M1-M4 keys cause a wake-up.
+    ite-device.wakeup.enable = (
+      mkEnableOption "Enable power wakeup on the internal USB keyboard-like device (8295 ITE Device) on Zephyrus GA402X"
+    ) // { default = false; };
```
Comment (matdibu) [inline asus/zephyrus/ga402x/shared.nix:40]: `mkEnableOption` already defaults to false, right?

Comment (Mic92) [reply asus/zephyrus/ga402x/shared.nix:40]: It does.

Comment (Mic92) [reply asus/zephyrus/ga402x/shared.nix:40]: https://github.com/NixOS/nixos-hardware/pull/1156

**Pull request merged**